examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,123 @@
+"""
+Copyright 2023 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import datetime
+import boto3
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+TABLE_NAME = os.getenv("TABLE_NAME")
+if not TABLE_NAME:
+    TABLE_NAME = input("DynamoDB events table name: ")
+
+aws_region = os.getenv("AWS_REGION")
+if not aws_region:
+    aws_region = input("AWS Region for DynamoDB: ")
+
+# Check ENV for start and end times, if none, set last 30 days
+start_date = os.getenv("START_DATE")
+if not start_date:
+    now = datetime.datetime.now()
+    start_date = now - datetime.timedelta(days=30)
+else:
+    start_date = datetime.datetime.strptime(start_date,'%Y-%m-%d')    
+
+end_date = start_date + datetime.timedelta(days=30)
+
+# Configure DynamoDB for query
+dynamodb = boto3.resource("dynamodb", region_name=aws_region)
+table = dynamodb.Table(TABLE_NAME)
+
+# variables for protocols
+app = 'app.session.start'
+db = 'db.session.start'
+kube = 'kube.request'

```

Comment:
Will this also pick up `kubectl exec` sessions?

Reply:
In Teleport 10, yes. In Teleport 11, first triggers a GET kube.request on the pod so yes it is captured, but then the exec sessions actually show up just like an SSH session start which may lead to an overcount on SSH as they use the same event type.

Reply:
Refactored the script, and corrected the double count issue by exempting event type `session.start` where there is a value for `kubernetes_cluster` in the item.